src/@next/globalStyles/constants.ts

```diff
@@ -0,0 +1,36 @@
+// colors
+export const autofillColor = "rgb(250, 255, 189)";
+export const autofillColorSelected = "rgb(232, 240, 254)";
+export const baseFontColor = "#323232";
+export const black = "#000";
+export const blue = "#21125e";
+export const blueDark = "#190c4e";
+export const blueOverlay = `rgba(${blue}, 0.15)`;
+export const gray = "#c4c4c4";
+export const grayDark = "#7d7d7d";
+export const grayLight = "#f6f6f6";
+export const green = "#3ed256";
+export const rose = "#c22d74";
+export const turquoise = "#51e9d2";
+export const turquoiseDark = "#55c4b3";
+export const turquoiseLight = "#f1f5f5";
+export const white = "#fff";
+
+// typography
+export const baseFontFamily = "montserrat";
+export const baseFontSize = "1rem"; // 16px
```

Comment:
it would be better to change root font-size to 62.5%.  It's easier to calculate px to rem (divide by 10).
But probably it's too late for that.

Reply:
While I agree the 62.5% rule is the gold standard, as you pointed out it might be too late for that as we are mostly copy-pasting styles from the "old" version and it could be time-consuming to recalculate the sizes everywhere else.